include/psa/crypto.h

```diff
@@ -104,6 +104,10 @@ extern "C" {
  * \retval #PSA_ERROR_HARDWARE_FAILURE
  * \retval #PSA_ERROR_CORRUPTION_DETECTED
  * \retval #PSA_ERROR_INSUFFICIENT_ENTROPY
+ * \retval #PSA_ERROR_STORAGE_FAILURE
+ * \retval #PSA_ERROR_DATA_INVALID
+ * \retval #PSA_ERROR_DATA_CORRUPT
+ * \retval #PSA_ERROR_INVALID_ARGUMENT
```

Comment:
It it coming from the `psa_crypto_load_transaction` and `psa_its_get` function if the transactions are enabled. The `INVALID_ARGUMENT` is not handled so it can happen if the saved transaction cannot be read, although it is very unlikely.